service/internal/auth/casbin.go

```diff
@@ -45,35 +45,35 @@ var defaultPolicy = `
 ## gRPC routes
 p,	role:org-admin,		policy.*,																*,			allow
 p,	role:org-admin,		kasregistry.*,													*,			allow
-p,	role:org-admin,		kas.AccessService/Rewrap, 			            *,			allow
+p,	role:org-admin,		kas.AccessService/Rewrap, 			        *,			allow
+p,  role:org-admin,   authorization.*,                        *,      allow
 ## HTTP routes
 p,	role:org-admin,		/attributes*,														*,			allow
 p,	role:org-admin,		/namespaces*,														*,			allow
 p,	role:org-admin,		/subject-mappings*,											*,			allow
 p,	role:org-admin,		/resource-mappings*,										*,			allow
 p,	role:org-admin,		/key-access-servers*,										*,			allow
-p,	role:org-admin, 	/kas/v2/rewrap,						  		*,      allow
-p,	role:org-admin,		/unsafe*,										*,			allow
+p,	role:org-admin, 	/kas/v2/rewrap,						  		        *,      allow
+p,	role:org-admin,		/unsafe*,										            *,			allow
 
 # Role: Admin
 ## gRPC routes
-p,	role:admin,		policy.*,																		*,			allow
-p,	role:admin,		kasregistry.*,															*,			allow
-p,	role:admin,		kas.AccessService/Info,					            *,			allow
-p,	role:admin,		kas.AccessService/Rewrap, 			            *,			allow
+p,	role:admin,		    policy.*,																*,			allow
+p,	role:admin,		    kasregistry.*,													*,			allow
+p,	role:admin,		    kas.AccessService/Rewrap, 			        *,			allow
+p,  role:org-admin,   authorization.*,                        *,      allow
 ## HTTP routes
 p,	role:admin,		/attributes*,																*,			allow
 p,	role:admin,		/namespaces*,																*,			allow
 p,	role:admin,		/subject-mappings*,													*,			allow
 p,	role:admin,		/resource-mappings*,												*,			allow
 p,	role:admin,		/key-access-servers*,												*,			allow
-p,	role:admin,		/kas/v2/rewrap,						  				*,      allow
+p,	role:admin,		/kas/v2/rewrap,						  				        *,      allow
```

Comment:
should we just make rewrap a public endpoint instead?

Reply:
We still need the access token validated and some information extracted for KAS.